feat(docs): add XRPL EVM Sidechain Axelar docs (#7)
ripple/docs.xrplevm.org: docs/evm-sidechain/axelar-interchain-evm-sidechain-xrpl.md

---
html: axelar-interchain-transfer-evm-sidechain-xrpl.html
blurb: Axelar interchain transfer to XRPL.
labels:
  - Interoperability
status: not_enabled
---
# Axelar Interchain Transfer from XRPL

> ## Using Portal

Reviewer: Remove this section header, unless there are plans for alternative bridging methods.

You can bridge native XRP from XRPL to XRPL EVM sidechain using Axelar Portal. In this example, we will transfer 5 XRP to an account in the XRPL EVM Sidechain.

> First, set XRPL as the source chain and XRPL EVM sidechain as the destination chain.

Reviewer: ```suggestion
1. Set XRPL as the source chain and XRPL EVM sidechain as the destination chain.
```

![portal set chains](../img/axelar-xrpl-set-chains.png)

> Then, select the faucet account on the XRPL chain.

Reviewer: ```suggestion
2. Select the faucet account on the XRPL chain.
```

![portal set chains](../img/axelar-xrpl-select-faucet.png)

> Then, connect your wallet using Metamask for the XRPL EVM Sidechain.

Reviewer: ```suggestion
3. Connect your wallet using Metamask for the XRPL EVM Sidechain.
```

![portal connect metamask](../img/axelar-xrpl-connect-metamask.png)

> Once your wallet is connected, you can start choosing the token you want to transfer and set the desired amount (currently, only XRP is supported). Click on `Transfer` and confirm the transaction in your wallet.

Reviewer: ```suggestion
4. Choose the token you want to transfer and set the desired amount (currently, only XRP is supported).

5. Click on `Transfer` and confirm the transaction in your wallet.
```

![portal transfer](../img/axelar-xrpl-wait-faucet-transaction.png)

> Then, await the transaction to be executed.

Reviewer: ```suggestion
```

> ![portal transaction executed](../img/axelar-xrpl-await-transaction.png)

Reviewer: ```suggestion
```
